cli/cmd/install.go

```diff
@@ -118,7 +81,6 @@ spec:
       annotations:
         conduit.io/created-by: "{{.CliVersion}}"
     spec:
-      serviceAccount: conduit-controller
```

Comment:
Thanks for backing out the other change. Looks like you've accidentally reverted a commit from #40 as part of this change.